builtin/providers/opsgenie/resource_opsgenie_team.go

```diff
@@ -53,10 +58,12 @@ func resourceOpsGenieTeamCreate(d *schema.ResourceData, meta interface{}) error
 	client := meta.(*OpsGenieClient).teams
 
 	name := d.Get("name").(string)
+	description := d.Get("description").(string)
 
 	createRequest := team.CreateTeamRequest{
-		Name:    name,
-		Members: expandOpsGenieTeamMembers(d),
+		Name:        name,
+		Description: description,
+		Members:     expandOpsGenieTeamMembers(d),
```

Comment:
yep - it's caught by the HCL (which picks it up as a null value) - there's also tests for omitting/empty/completed values